.github/runner/set-k8s-secrets.sh

```diff
@@ -108,7 +108,7 @@ VRO_SECRETS_NAMES="VRO_SECRETS_API VRO_SECRETS_SLACK VRO_SECRETS_MAS VRO_SECRETS
 # and evaluated by set-env-secrets.src in each VRO container.
 # Advantage: New environment variables can be added to these secrets without modifying
 # Helm configurations -- simply add them to Vault.
-SECRET_DATA=$(collectSecretExportCmds $VRO_SECRETS_NAMES)
+SECRET_DATA=$(collectSecretExportCmds "$VRO_SECRETS_NAMES")
```

Comment:
Undo this change so that the values in `VRO_SECRETS_NAMES` are treated as separate arguments. 
```suggestion
SECRET_DATA=$(collectSecretExportCmds $VRO_SECRETS_NAMES)
```

Adding quotes treats it as a single argument:
```sh
 > VRO_SECRETS_NAMES="VRO_SECRETS_API VRO_SECRETS_SLACK"
 > hello(){ echo "$1, $2"; }
 > hello "$VRO_SECRETS_NAMES"
VRO_SECRETS_API VRO_SECRETS_SLACK,

 > hello $VRO_SECRETS_NAMES
VRO_SECRETS_API, VRO_SECRETS_SLACK
```